### ISSUE TITLE
Discuss: Should fast-startup synced with upstream to make adoptium pipeline work?

### ISSUE THREAD

**Issue by kelthuzadx** [adoptium/jdk11u-fast-startup-incubator#36]
I created an adoptium pipeline for fast-startup project. The original purpose of this pipeline is to track every new patch we submitted, to ensure they work well. Recently, this pipeline has been failing for a while, the reason is that it tries keeping up to date with upstream automatically but fails to do so due to merge conflict. @jianglizhou  Should we keep synced with upstream? Do you have any comments? If so, I'd like to merge upstream later.

**Comment by jianglizhou** [comment]
Keeping synced with upstream for the fast-startup project is a good idea. Does the pipeline integrate automatically? 

If we manually sync [jdk11u-fast-startup-incubator](https://github.com/adoptium/jdk11u-fast-startup-incubator) with OpenJDK 11 and manually resolve the conflicts, would that help the pipeline process?

**Comment by kelthuzadx** [comment]
> Keeping synced with upstream for the fast-startup project is a good idea. Does the pipeline integrate automatically?
> 
> If we manually sync [jdk11u-fast-startup-incubator](https://github.com/adoptium/jdk11u-fast-startup-incubator) with OpenJDK 11 and manually resolve the conflicts, would that help the pipeline process?

Yes, after manual conflict resolution, this pipeline could fetch upstream changes and merge automatically.
